dapp-test.json

```diff
@@ -24,6 +24,7 @@
     "src/Deposit.sol": { "urls": ["src/Deposit.sol"] },
     "src/ValidatorsDiamond.sol": { "urls": ["src/ValidatorsDiamond.sol"] },
     "src/QueueLibrary.t.sol": { "urls": ["src/QueueLibrary.t.sol"] },
+    "src/facets/ValidatorLocationsLibrary.t.sol": { "urls": ["src/facets/ValidatorLocationsLibrary.t.sol"] },
```

Comment:
@kasvtv The test files need to be referenced in dapp-test.json so the tests will run. I updated so this is just an FYI.